target/linux/brcmiproc/image/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2016-2017 Yousong Zhou <yszhou4tech@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+define Device/Default
+  DEVICE_DTS_DIR := $(DTS_DIR)/broadcom
```

Comment:
I think it would be better if you defined
DTS_DIR := $(DTS_DIR)/broadcom
outside this node.

DTS_DIR is global, DEVICE_DTS_DIR may be different per device (also see below).

Reply:
Defined DTS_DIR as suggested